stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/startup/ThreeTenInitializer.kt

```diff
@@ -0,0 +1,14 @@
+package io.getstream.chat.android.ui.startup
+
+import android.content.Context
+import androidx.startup.Initializer
+import com.jakewharton.threetenabp.AndroidThreeTen
+
+@Suppress("unused")
+internal class ThreeTenInitializer : Initializer<Unit> {
```

Comment:
Wouldn't it be enough to have one of these in the old UI module?

Reply:
Ups, I forgot to remove this one

Reply:
Are we eventually migrating everything over to `ui-components` at some point? Does this become something we would eventually migrate back?

Reply:
We can push this date formatting code down into the upcoming `stream-chat-android-ui-common`, it can live there. I don't think we'll move it up to `ui-components` at any point.